projects/Mallard/src/components/onboarding/onboarding-card.tsx

```diff
@@ -68,63 +69,83 @@ const OnboardingCard = ({
-                        getFont('titlepiece', size === 'big' ? 2.5 : 2.0),
-                        { marginBottom: size === 'big' ? 16 : 8 },
-                        appearances[appearance].text,
-                    ]}
-                >
-                    {title}
-                </TitlepieceText>
-                {subtitle && (
+}) => {
+    const max = minScreenSize() * 0.8
+    return (
+        <View
+            style={[
+                appearances[appearance].background,
+                {
+                    maxHeight: max,
+                    maxWidth: max,
+                    aspectRatio: 1,
+                    flexDirection: 'column',
+                },
+                style,
+            ]}
+        >
+            <View style={[styles.top, appearances[appearance].background]}>
+                <View style={{ flexGrow: 1 }}>
                     <TitlepieceText
+                        accessibilityRole="header"
                         style={[
-                            getFont('titlepiece', size === 'big' ? 1.5 : 1.25),
+                            getFont('titlepiece', size === 'big' ? 2.5 : 2.0),
+                            { marginBottom: size === 'big' ? 16 : 8 },
                             appearances[appearance].text,
                         ]}
                     >
-                        {subtitle}
+                        {title}
                     </TitlepieceText>
-                )}
-            </View>
-            <View>
-                {mainActions && (
-                    <View style={{ flexDirection: 'row', flexWrap: 'wrap' }}>
-                        {mainActions.map(({ label, onPress }) => (
-                            <Button
-                                style={{
-                                    marginTop: 10,
-                                    marginRight: 10,
-                                }}
-                                appearance={ButtonAppearance.light}
-                                key={label}
-                                onPress={onPress}
-                            >
-                                {label}
-                            </Button>
-                        ))}
-                    </View>
-                )}
+                    {subtitle && (
+                        <TitlepieceText
+                            style={[
+                                getFont(
+                                    'titlepiece',
+                                    size === 'big' ? 1.5 : 1.25,
+                                ),
+                                appearances[appearance].text,
+                            ]}
+                        >
+                            {subtitle}
+                        </TitlepieceText>
+                    )}
+                </View>
+                <View>
+                    {mainActions && (
+                        <View
+                            style={{ flexDirection: 'row', flexWrap: 'wrap' }}
```

Comment:
As above.

Reply:
There were already there but happy to pull them out 👍